files/Dockerfile.tester

```diff
@@ -0,0 +1,72 @@
+FROM nginx:latest
+
+COPY tests/test_data/runfolders /data/tester/runfolders
```

Comment:
Does the source refer to the snpseq_packs repo? Do we have minimal runfolders there?

Reply:
yes, that's the context specified in `docker_compose.yml`. I have a PR coming with a mock runfolder (and instructions)